test/generator/template_values_test.rb

```diff
@@ -2,26 +2,44 @@
 
 module Generator
   class TestCasesValuesTest < Minitest::Test
+    def setup
+      @arguments = {
+        abbreviated_commit_hash: nil, version: nil, exercise_name: nil, test_cases: nil
+      }
```

Comment:
I see what you're doing here, but it's plastering over a big crack that is appearing.
I'd rather not do it and feel the pain in every test and hope that inspires fixing the crack that is an ever expanding argument list.